Shopipy.Web/ViewModels/HomeViewModel.cs

```diff
@@ -0,0 +1,3 @@
+namespace Shopipy.Web.ViewModels;
+
+public class HomeViewModel : BaseViewModel {}
```

Comment:
Probably not much reason to have it if it's empty now

Reply:
BaseViewModel is abstract, can't use it directly.